src/WSRequestHandler_General.cpp

```diff
@@ -231,6 +231,45 @@ HandlerResponse WSRequestHandler::HandleGetStats(WSRequestHandler* req) {
 	return req->SendOKResponse(response);
 }
 
+/**
+ * Broadcast some Data (String) to all connected Websocket-Clients
+ *
+ * @param {String} `realm` Some Identifier to be choosen by the client
+ * @param {String} `data` User-defined data String
```

Comment:
It'd be better to make it an object instead.

```suggestion
 * @param {Object} `data` User-defined data object
```